tests/js/panelsTests.js

```diff
@@ -362,6 +345,70 @@ https://github.com/gpii/universal/LICENSE.txt
         jqUnit.assertEquals("The decrease button should have the correct enabled/disabled state", decreaseDisabled, that.locate("decrease").prop("disabled"));
     };
 
+    /*************************
+     * Text Size Panel Tests *
+     *************************/
+
+    fluid.defaults("gpii.tests.firstDiscovery.panel.textSize", {
+        gradeNames: ["gpii.firstDiscovery.panel.textSize", "autoInit"],
+        messageBase: {
+            rangeInstructions: "Adjust the text and controls to a size you like best.",
+            maxLabel: "max",
+            minLabel: "min",
+            increaseLabel: "larger",
+            decreaseLabel: "smaller"
+        },
+        model: {
+            value: 1
+        },
+        modelListeners: {
+            // rerenders on modelChange like panel behaves in the prefsEditor
+            "value": "{that}.refreshView"
+        }
+    });
+
+    fluid.defaults("gpii.tests.textSizePanel", {
+        gradeNames: ["gpii.tests.rangePanel", "autoInit"],
+        components: {
+            range: {
+                type: "gpii.tests.firstDiscovery.panel.textSize",
+                container: ".gpiic-fd-textSize"
+            }
+        }
+    });
+
+    /***************************
+     * Speech Rate Panel Tests *
+     ***************************/
+
+    fluid.defaults("gpii.tests.firstDiscovery.panel.speechRate", {
+        gradeNames: ["gpii.firstDiscovery.panel.speechRate", "autoInit"],
+        messageBase: {
+            rangeInstructions: "Adjust the speed at which items on the screen are read out loud.",
+            maxLabel: "fast",
+            minLabel: "slow",
+            increaseLabel: "faster",
+            decreaseLabel: "slower"
+        },
+        model: {
+            value: 1
+        },
+        modelListeners: {
+            // rerenders on modelChange like panel behaves in the prefsEditor
+            "value": "{that}.refreshView"
+        }
+    });
+
+    fluid.defaults("gpii.tests.speechRatePanel", {
+        gradeNames: ["gpii.tests.rangePanel", "autoInit"],
```

Comment:
I think it's unnecessary to run the same set of the rangePanel tests for every panel that makes use of the rangePanel since the derived panels are pretty much the same as the rangePanel itself but with preferenceMap defined. We could test the rangePanel and only test derived panels when they perform extra functionalities. What do you think?